src/main/java/net/seesharpsoft/intellij/plugins/csv/editor/table/swing/CsvTableEditorSwing.java

```diff
@@ -189,8 +198,10 @@ public void updateEditorLayout() {
 
         for (int i = 0; i < currentColumnCount; ++i) {
             TableColumn column = this.tblEditor.getColumnModel().getColumn(i);
-            column.setPreferredWidth(columnWidths[i]);
-            column.setWidth(columnWidths[i]);
+            int width = columnWidths[i];
+            width = width * getRowLineHeight() / BASE_ROW_LINE_HEIGHT;
```

Comment:
With respect to the discussion about persisting font-size or not: if it is not persisted then the column-widths should be indeed also adjusted on the fly only and not changed in the editor state. Then you do not have the change-rate and you need to stick with a fix anchor size - as you are making use of `EditorColorsManager.getInstance().getGlobalScheme().getEditorFontSize()` as default font-size, I suppose that this value needs to be used here as well (instead of `BASE_ROW_LINE_HEIGHT`).